timelock-server/src/main/java/com/palantir/atlasdb/timelock/logging/NonBlockingFileAppenderFactory.java

```diff
@@ -28,6 +30,7 @@
  * appender. This prevents request log rollovers from blocking request threads.
  * TODO(nziebart): remove this when we switch to internal web framework
  */
+@AutoService(Discoverable.class)
```

Comment:
Instead of explicitly calling `registerSubtype`, we just leverage the auto discoverable mechanism that Dropwizard uses.